src/TensorSharp/TensorImageExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class TensorImageExtensions
         /// </summary>
         /// <param name="tensor"></param>
         /// <returns></returns>
-        public static Bitmap ToBitmap(this Tensor tensor)
```

Comment:
Since this file is heavily dependent on Windows, I would suggest just removing the whole file. The only alternative I can think of would be to use a portable library like https://github.com/SixLabors/ImageSharp for the bitmap representation. However, I think there is an argument for this functionality being outside the remit of this library, so I don't have any problem with just removing this file.

Reply:
Agree to remove, should reside outside of core library.